photonai_neuro/nifti_transformations.py

```diff
@@ -101,17 +99,26 @@ def __init__(self, voxel_size: Union[int, List] = 3, interpolation: str = 'neare
         super(ResampleImages, self).__init__(output_img=output_img)
         self._voxel_size = None
         self.voxel_size = voxel_size
+        self._interpolation = None
+        self.interpolation = interpolation
+
+
+    def fit(self, X, y=None, **kwargs):
+        return self
 
-        if interpolation in ['continuous', 'linear', 'nearest']:
```

Comment:
This was done in init, however if some changes it after isntance creation, this validity check will no longer be done, so I created a property.